components/serializer.rst

```diff
@@ -573,6 +573,52 @@ having unique identifiers::
     echo $serializer->serialize($org, 'json');
     // {"name":"Les-Tilleuls.coop","members":[{"name":"K\u00e9vin", organization: "Les-Tilleuls.coop"}]}
 
+Handling Arrays
+---------------
+
+The serializer component is capable of handling arrays of objects as well.
+Serializing such an array works just like serializing a single object:
+
+.. code-block:: php
```

Comment:
as the previous sentence ended with a colon, we can use (and we do always) the shorthand (double colon):

```rst
Serializing arrays works just like serializing a single object::

    $person1 = new Acme\Person();
    // ...
```